core/src/main/java/nl/nn/adapterframework/batch/IRecordHandlerManager.java

```diff
@@ -38,6 +39,7 @@
 	/**
 	 * @param flow New flow to be added to the managed flow elements
 	 */
+	@IbisDoc("flow to be added to the managed flow elements")
```

Comment:
This description starts with a lower-caps letter. How about starting with a capital to form a sentence?

Reply:
I do not see a number here, so no order is specified here yet for the XSD. Do you want that the XSD requests the same order here as the order of the methods?